src/DataBus/ConfigureAzureDataBus.cs

```diff
@@ -150,10 +156,35 @@ public static DataBusExtensions<AzureDataBus> CleanupInterval(this DataBusExtens
             {
                 throw new ArgumentOutOfRangeException(nameof(cleanupInterval), cleanupInterval, "Should not be negative.");
             }
+
             GetSettings(config).CleanupInterval = cleanupInterval;
             return config;
         }
 
+        /// <summary>
+        ///  Sets token credential to authenticate with Storage Blob service
+        /// <remarks>Token credentials can be created using <see cref="AzureServiceTokenProvider"/> with token renewal configured.</remarks>
+        /// </summary>
+        public static DataBusExtensions<AzureDataBus> AuthenticateWithManagedIdentity(this DataBusExtensions<AzureDataBus> config, string storageAccountName, TimeSpan renewalTimeBeforeTokenExpires)
+        {
+            if (renewalTimeBeforeTokenExpires <= TimeSpan.Zero)
```

Comment:
Using `<=` here but `<` ]above](https://github.com/Particular/NServiceBus.DataBus.AzureBlobStorage/pull/91/files#diff-92584a4098652b2fd300a137278c018fR74). Does the difference matter?

Reply:
`TimeSpan.Zero` as a value for the time to start refresh before the token expires doesn't make a lot of sense as the token will be lost at that point, no?